sky/serve/load_balancer.py

```diff
@@ -223,10 +229,25 @@ async def startup():
         uvicorn.run(self._app, host='0.0.0.0', port=self._load_balancer_port)
 
 
-def run_load_balancer(controller_addr: str, load_balancer_port: int):
-    load_balancer = SkyServeLoadBalancer(controller_url=controller_addr,
-                                         load_balancer_port=load_balancer_port)
-    load_balancer.run()
+def run_load_balancer(controller_addr: str,
+                      load_balancer_port: int,
+                      policy_name: Optional[str] = None):
+    """ Run the load balancer.
+
+    Args:
+        controller_addr: The address of the controller.
+        load_balancer_port: The port where the load balancer listens to.
+        policy_name: The name of the load balancing policy to use. Defaults to
+            None.
+    """
+    try:
+        load_balancer = SkyServeLoadBalancer(
+            controller_url=controller_addr,
+            load_balancer_port=load_balancer_port,
+            load_balancing_policy_name=policy_name)
+        load_balancer.run()
+    except ValueError as e:
+        logger.error(f'Failed to start load balancer: {e}')
```

Comment:
we probably dont want to silently allow the failure to happen. Remove the try-except block?

Reply:
agreed